rusk-wallet/src/wallet.rs

```diff
@@ -449,22 +399,11 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
             return Err(Error::NotEnoughGas);
         }
 
-        let mut rng = StdRng::from_entropy();
-        let sender_index =
-            sender.index().expect("owned address should have an index");
+        // let mut rng = StdRng::from_entropy();
+        // let sender_index = sender.index()?;
+        // let sender_sk = derive_phoenix_sk(&self.seed, sender_index);
```

Comment:
left-over comments? And where are you generating the transaction?

Reply:
Fixed `execute`